Configuration/Generator/python/concurrentLumisDisable.py

```diff
@@ -5,6 +5,7 @@
     "CosMuoGenProducer",
     "ExhumeGeneratorFilter",
     "Herwig7GeneratorFilter",
+    "Herwig7HadronizerFilter",
```

Comment:
@Dominic-Stafford this file is not in the master branch

Reply:
Hi Meng, this file is in master (and all releases since CMSSW_12). This line is the one change in #42732 as proposed here: https://github.com/cms-sw/cmssw/pull/42673#issuecomment-1708416500